identity/roles.go

```diff
@@ -189,3 +190,81 @@ func (c *RolesClient) Delete(ctx context.Context, input *DeleteRoleInput) error
 
 	return nil
 }
+
+// RoleTags
```

Comment:
comment on exported type RoleTags should be of the form "RoleTags ..." (with optional leading article)

Reply:
All the comments are coming right up! Thank you! 👍